test/timing_load_creds.c

```diff
@@ -174,10 +183,18 @@ int main(int ac, char **av)
         perror("elapsed start");
         exit(EXIT_FAILURE);
     }
+# if defined(OPENSSL_SYS_TANDEM)
+    times(&rus);
+    u_start.tv_sec = 0;
+    u_start.tv_usec = rus.tms_utime;
+    s_start.tv_sec = 0;
+    s_start.tv_usec = rus.tms_stime;
```

Comment:
I don't think this math is quite right.  times returns a structure that represents the elapsed cpu time in clock ticks, and this will be wrong if the run time has gone on for more than a second.  I think what you want is this math:

```
u_start.tv_sec = rus.tms_utime / CLOCKS_PER_SEC;
u_start_tv_usec = ((rus.tms_utime * CLOCKS_PER_SEC) - rus.tms_utime) * 1000000;
```

Or something like that.  Basically you need to convert the 64 bit cpu tick counter to a number of seconds and nanoseconds for the start and end user and system clock values.

Reply:
Please have a look at the changes. I changed the #if silliness to depend on RUSAGE_SELF instead of NONSTOP to cover more situations.